components/search/SearchCollection.vue

```diff
@@ -58,7 +58,7 @@ import KeyboardEventsMixin from '~/utils/mixins/keyboardEventsMixin'
     BasicSwitch: () => import('@/components/shared/form/BasicSwitch.vue'),
   },
 })
-export default class CollectionSearchBar extends mixins(KeyboardEventsMixin) {
+export default class extends mixins(KeyboardEventsMixin) {
```

Comment:
You miss class name here. It will emit error in Vue

Reply:
Added